### ISSUE TITLE
How to visualize the proportional distribution of positive and negative sample gradients of different category classifiers

### ISSUE THREAD

**Issue by sanmulab** [open-mmlab/mmdetection#6286]
Hello, I want to ask if there is any visual code or tool script for the gradient proportional distribution of positive and negative samples of different categories of classifiers in mmdet. For example, the visualization of the gradient ratio of positive and negative samples in **Seesaw Loss** paper. I also want to have such visual pictures added to my paper. Thank you for telling me.

**Comment by hhaAndroid** [comment]
@chensen668 Sorry, we do not provide such tools at the moment.

**Comment by chensen668** [comment]
> @chensen668 Sorry, we do not provide such tools at the moment.

Is there any relevant instance code? I think some authors say that  can collect positive and negative gradients (of output Logits) in the training for each category. But I don't know how to encode it.
